feature-group-definitions/scroll-to-text-fragment.yml

```diff
@@ -0,0 +1,7 @@
+name: Scroll to text fragment
+description: Scrolling to a text snippet given in a URL fragment such as `#:~:text=snippet`.
```

Comment:
Reviewers, I don't love this description, but can't figure out how to write it in a style more similar to other descriptions. Naming the feature via the syntax is tricky here, and saying what the syntax does is also tricky. Something like this?

> A URL fragment on the form `#:~:text=snippet` causes the page to scroll to the snippet of text, if found.

Reply:
"fragment" in `name` reads as equivalent to "snippet" in `description`, but I think it's supported to refer to the segment of a URL named a fragment, so that's a bit confusing.

also, "scroll to" feels overspecific - the spec talks about "emphasizing" or "bringing to the user's attention"; I'm not sure we would call the id fragment of HTML "scroll to id fragment".

How about
```yaml
name:  Link to text snippet
description: A URL fragment on the form `#:~:text=snippet` allows to link directly to the snippet of text, if found.
```
"Link to text snippet" could also be "Text snippet URL fragment", not sure how widely known the term "url fragment" is

Reply:
I agree that "scroll to" isn't the whole story, as implemented in Chrome the text is also highlighted. I'm a bit wary of picking a name that's never been used before however. Both https://developer.mozilla.org/en-US/docs/Web/Text_fragments and https://web.dev/articles/text-fragments both call this "text fragments", perhaps that would be best?

Reply:
About the description, here's MDN:

> Text fragments allow linking directly to a specific portion of text in a web document, without requiring the author to annotate it with an ID, using particular syntax in the URL fragment. Supporting browsers are free to choose how to draw attention to the linked text, e.g. with a color highlight and/or scrolling to the content on the page.

And here's web.dev:

> Text Fragments let you specify a text snippet in the URL fragment. When navigating to a URL with such a text fragment, the browser can emphasize and/or bring it to the user's attention.

I'll try something...

Reply:
Tried.

Reply:
The new description looks good; I'm not convinced by the name - it feels too generic (although I appreciate the precedent set by MDN/web.dev)

Reply:
I agree it's a bit generic, but what to call it? Link to text fragment?

Reply:
The "link to" feels a bit... extra. I don't think we would call the age-old `#id` syntax "Link to fragment identifier", it's a part of a URL so of course it can be linked.